netci.groovy

```diff
@@ -1251,6 +1251,11 @@ combinedScenarios.each { scenario ->
                     // The tuples (LinuxARMEmulator, other architectures) are not handled and control returns
                     def isLinuxEmulatorBuild = false
                     if (os == 'LinuxARMEmulator' && architecture == 'arm') {
+                        // Cross Builds only in Debug and Release modes allowed
+                        if ( configuration == 'Checked' ) {
```

Comment:
Nit: Would you mind changing this to lowerConfiguration == 'checked' to be consistent with the rest of the checks.